### ISSUE TITLE
while navigating from other pages sometimes the styles are off for certain elements. 

### ISSUE THREAD

**Issue by nisarhassan12** [gitpod-io/website#1247]
derived from https://github.com/gitpod-io/website/pull/1131#issuecomment-968605968

![image](https://user-images.githubusercontent.com/46004116/141838672-1c79dfaa-2249-476b-ac76-d681fadbc67c.png)

**Comment by nisarhassan12** [comment]
@mikenikles I have encountered this again today while navigating from another page to pricing. I tried opening the console and the network tab to check if there is some CSS requested failing but there is nothing there.  

![image](https://user-images.githubusercontent.com/46004116/142358558-3170dce6-625e-4bf9-b8c1-4a94fa1fa10b.png)

I'm able to reproduce this, again and again, every time on Chrome and Firefox if I first navigate to `www.gitpod.io/` the homepage and then click in the nav links in a sequence and eventually get to the pricing page and scroll down to the FAQS. 

@ChristinFrohne Also encountered this again here https://github.com/gitpod-io/website/issues/1253#issue-1054715854
